src/NUnitFramework/tests/Attributes/RepeatableTestsWithTimeoutAttributesTests.cs

```diff
@@ -90,21 +90,81 @@ public void ShouldFailOnMultipleRepeatableAttributesIncludingCustom()
             Assert.AreEqual("Invalid", result.ResultState.Label);
         }
 
+        [Test]
+        public void ShouldFailWithRepeatableAndSetupTearDownCustomAttributes()
+        {
+            CustomRepeaterAndSetupAndTearDown.AttributeCallCount = 0;
+            var testCase = TestBuilder.MakeTestCase(GetType(), nameof(TestMethodForRepeatableAndSetupTearDownCustomAttribute));
+            var workItem = TestBuilder.CreateWorkItem(testCase);
+            TestBuilder.ExecuteWorkItem(workItem);
+
+            Assert.That(CustomRepeaterAndSetupAndTearDown.AttributeCallCount, Is.EqualTo(2));
```

Comment:
The title of the test is "ShouldFail" but I'm not seeing where a failure is tested for. Does it make any sense for the attribute to wrap the command at two different levels? I don't think so, because it can't know which of the two reasons `Wrap` is being called.

Reply:
I only wrote that test last night and I can't understand what I was thinking a case was... I have removed it.

Reply:
I think this is actually a valuable test and we should keep testing it. I was wondering though what the test outcome should be. I think we need a new error mode where we mark the test invalid if the same class implements `IRepeatTest` and `IWrapSetUpTearDown`. The test would be the same as the above test, but you would assert that the test case is marked as not runnable.